flows/classifiers.go

```diff
@@ -0,0 +1,37 @@
+package flows
+
+import "github.com/nyaruka/goflow/assets"
+
+// Classifier represents an NLU classifier.
+type Classifier struct {
```

Comment:
should this be `ClassifierNLU` if the action is `ClassifyNLU`? (to be clear my preference is to drop the nlu in both)